app/services/emotion_diary_service.py

```
@@ -0,0 +1,37 @@
+            APIException: GPT 응답 실패 시
+        """
+        try:
+            result = GPTService.analyze_emotion_diary(content)
+            return EmotionDiaryResponse(
+                emotion=result["emotion"],
+                summary=result["summary"],
+                feedback=result["feedback"]
+            )
+        except Exception as e:
+            logger.error(f"[EmotionDiaryService] 감정 분석 실패: {e}")
+            raise APIException(ErrorStatus.GPT_API_ERROR)
```

Ruff (0.12.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)